runtime/hydradx/src/lib.rs

```diff
@@ -773,13 +773,8 @@ impl_runtime_apis! {
 				}
 
 				fn reserve_transferable_asset_and_dest() -> Option<(Asset, Location)> {
-					Some((
-						Asset {
-							fun: Fungible(ExistentialDeposit::get()),
-							id: AssetId(Parent.into())
-						},
-						ParentThen(Parachain(RandomParaId::get().into()).into()).into(),
-					))
+					// TODO: https://github.com/galacticcouncil/HydraDX-node/issues/840
```

Comment:
since we won't solve this in the upgrade, maybe you could refer to this so something like `//TODO: fix it in next upgrade` or so

Reply:
updated. There is also more info in the issue